ticdc/ticdc-sink-to-cloud-storage.md

```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
 
```

Comment:
`ticdc/ticdc-changefeed-config.md` in https://github.com/pingcap/docs-cn/pull/13654/files has been translated in a separate PR: https://github.com/pingcap/docs/pull/13357